CondCore/CondDB/test/condTestRegression.py

```diff
@@ -159,11 +169,12 @@ def runSelf(self, readOrWrite, dbNameIn=None):
           
           if readOrWrite == 'write':
               self.dbList['SELF'] = dbName
-
+              
           # we run in the local environment, but need to make sure that we start "top-level" of the devel area
           # and we assume that the test was already built 
+          payloadScript = 'test/%s/testReadWritePayloads' % self.arch
           cmd = 'export SCRAM_ARCH=%s; cd %s/src; eval `scram runtime -sh 2>/dev/null` ; ' % (os.environ['SCRAM_ARCH'],os.environ['CMSSW_BASE'], )
```

Comment:
@ggovi , if CMSSW_BASE is set then this means cmsenv is already set, so in this case we do not need to source the env again .... rght?